src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+
+Your PWA can prompt the user to open a file, open mutiple files, open a directory and save files! Your PWA can also prompt the user for write access to an existing file, allowing your pwa to transparently write to a file without any extra downloads needed.
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a demo <a href="https://wiggly-impossible-crayfish.glitch.me/file-handling.html" target="_blank" rel="noopener">here</a> of the File System Access API!
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://web.dev/file-system-access/" target="_blank" rel="noopener">this tutorial</a> to learn how to use the File System Access API and enable this functionality in your PWA.
+  </p>
+</div>
+
+## Sharing
+
+Sharing content, and ingesting shared content, has always required custom UI and logic from the developer, until today! Your PWA can easily share content using the native Share UI built into Windows:
+
+<img src="/docs/windows/pwa-api-documentation/share.png" width="350px" alt="A screenshot of the share dialog on Windows" /> 
+
+<div class="demo-block">
+  <strong>Demo</strong>
+
+  <p>
+    Check out a demo <a href="https://wiggly-impossible-crayfish.glitch.me/web-share.html" target="_blank" rel="noopener">here</a> of the Web Share API!
+  </p>
+</div>
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    Check out <a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Web Share API and enable this functionality in your PWA
+  </p>
+</div>
+
+Your PWA can also register as a Share Target, enabling your PWA to have content shared directly to it.
+
+<div class="how-to">
+  <strong>How To: Share Target</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/webappmanifests#identify-your-app-as-a-share-target" target="_blank" rel="noopener">this tutorial</a> to learn how to use the Share Target API and enable this functionality in your PWA.
+  </p>
+</div>
+
+Finally, your PWA can use the early File Handlers API to register as a File Handler for certain file types. This enables you to open a file directly into a PWA, just as you can open images directly into the Photos app from the File Explorer:
+
+<div class="how-to">
+  <strong>How To: File Handling</strong>
+
+  <p>
+    Check out <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/experimental-features/#file-handling" target="_blank" rel="noopener">this documentation</a> to learn more!
+  </p>
+</div>
+
+## Shortcuts
+
+Using the Web Shortcuts API, PWAs can tie into the shortcuts UI in Windows, enabling users to jump to specific pages or actions in your PWA! This can be useful for
```

Comment:
"This can be useful for" -- this feels the sentence ended abruptly. Maybe add a colon at the end, e.g. "This can be useful for apps such as:"